[media-controls] add the exit fullscreen icon for iOS

graouts opened the pull request:
#### ee7bcbc5672177771432f9912fc64baa95366ebc
<pre>
[media-controls] add the exit fullscreen icon for iOS
<a href="https://bugs.webkit.org/show_bug.cgi?id=242505">https://bugs.webkit.org/show_bug.cgi?id=242505</a>
rdar://96618631

Reviewed by Dean Jackson.

* Source/WebCore/Modules/modern-media-controls/images/iOS/ExitFullscreen.svg: Added.

Canonical link: <a href="https://commits.webkit.org/252486@main">https://commits.webkit.org/252486@main</a>
</pre>

nt1m commented on Source/WebCore/Modules/modern-media-controls/images/iOS/ExitFullscreen.svg:
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="16px" height="16px" viewBox="0 0 16 16" version="1.1" xmlns="http://www.w3.org/2000/svg">
```
```suggestion
<svg width="16" height="16" viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg">
```

grorg replied:
Agreed to removing the `version` attribute. I'm not sure what the benefit of removing the XML declaration is - this is valid XML after all. 

As for removing the `px` unit specifiers from the `width` and `height`… I'm not sure it matters. It would mean the value is in the current coordinate system, but that would be the same as `px`. I think it might be better to simply remove the attributes completely, but they are actually used from the media controls code to evaluate the size of the button, iirc.

grorg replied:
I wonder if I can push to this branch on Antoine's repository.

grorg replied:
Actually all the SVG images in media controls have the version attribute and `px` width/height. I guess the files come from HI, so we should be consistent here. I'll add the merge label.

graouts committed:
[media-controls] add the exit fullscreen icon for iOS
https://bugs.webkit.org/show_bug.cgi?id=242505
rdar://96618631

Reviewed by Dean Jackson.

* Source/WebCore/Modules/modern-media-controls/images/iOS/ExitFullscreen.svg: Added.

Canonical link: https://commits.webkit.org/252486@main
